_layouts/nightly.html

```diff
@@ -157,17 +157,17 @@
                   <li><a dl="imagemap-plugin">Image Map</a></li>
                   <li><a dl="libjpeg-turbo-plugin">JPEG Turbo</a></li>
                   <li><a dl="netcdf-out-plugin">NetCDF</a></li>
-                  <li><a dl="ogr-plugin">OGR</a></li>
+                  <li>OGR (<a dl="ogr-wfs-plugin">WFS</a>, <a dl="ogr-wps-plugin">WPS</a>)</li>
                   <li><a dl="xslt-plugin">XSLT</a></li>
                 </ul>
               </div>
               <div class="col-sm-6">
                 <h4>Services</h4>
                 <ul>
                   <li><a dl="csw-plugin">CSW</a></li>
-                  <li><a dl="wcs-plugin">WCS 2.0</a></li>
```

Comment:
@jodygarnett WCS 2.0 is now in core and not a plugin. Only WCS 2.0 EO profile support (next line) is still a plugin .